qa/.ssh_config

```diff
@@ -0,0 +1 @@
+[{}]
```

Comment:
this file should not be committed, we should add it to the gitignore. I propose we add one for the qa level, easy to manage.